node/node.go

```diff
@@ -711,3 +711,13 @@ func (node *Node) SetSyncFreq(syncFreq int) {
 func (node *Node) SetBeaconSyncFreq(syncFreq int) {
 	node.beaconSyncFreq = syncFreq
 }
+
+// ShutDown gracefully shut down the node server and dump the in-memory blockchain state into DB.
+func (node *Node) ShutDown() {
```

Comment:
do we need to flush the disk?

Reply:
The Blockchain().Stop() will flush the state to disk, and also wait for all blockchain operations to fully finish (there is a wait group)